### ISSUE TITLE
Only shows 20 contacts on the left

### ISSUE THREAD

**Issue by g8f6r4e7x2** [ifedapoolarewaju/igdm#24]
I can only access 20 contacts on the left and not the older messages. On the Instagram app, you can keep scrolling down and it will load older messages. Thank you.

**Comment by ifedapoolarewaju** [comment]
Yes, this was initially low priority because you can search users with the search bar, but will add this to todo 👍

**Comment by g8f6r4e7x2** [comment]
Yes it is possible to search for the user but it does not bring the history of the chat if the user is not on the latest 20.

Thank you @ifedapoolarewaju

**Comment by alibekch** [comment]
requested feature @ifedapoolarewaju

**Comment by baptx** [comment]
@b42thomas is your issue related to this one? https://github.com/ifedapoolarewaju/igdm/issues/519

**Comment by baptx** [comment]
Since it is still not possible to view all conversations history in IG:dm (bugs #24 and #513), I made a script to backup the direct messages from a web browser: https://gist.github.com/baptx/1e61eef2e1ec200b6e7b32409f79c07f
Maybe it will help some people.
